enforcer/netfilter/netfilter_linux.go

```diff
@@ -231,6 +230,7 @@ func (nfq *NFQueue) run() {
 
 }
 
+// nolint : deadcode
 //export processPacket
```

Comment:
yeap .. linters think its dead, but its actually the most critical code of the whole package.